packages/ui/components/Menu.tsx

```diff
@@ -0,0 +1,74 @@
+import classnames from 'classnames'
+import Link from 'next/link'
+
+export interface MenuItem {
+  text: string
+  url: string
+}
+
+export type MenuVariant = 'primary' | 'secondary' | 'text'
+
+export interface MenuProps {
+  items: MenuItem[]
+  variant?: MenuVariant
+  size?: 'small' | 'regular' | 'large'
+  flexJustify?: 'start' | 'end' | 'center' | 'between' | 'around' | 'evenly'
+  flexGap?: number
+  containerClasses?: string
+}
+
+export const Menu = ({
+  items,
+  variant = 'primary',
+  size = 'regular',
+  flexJustify = 'start',
+  flexGap,
+  containerClasses,
+}: MenuProps) => {
+  const flexGapClass = flexGap && `gap-${flexGap}`
+
+  return (
+    <div
+      className={classnames(
+        'flex h-full w-full items-center',
+        flexGapClass,
+        {
+          'justify-start': flexJustify === 'start',
+          'justify-end': flexJustify === 'end',
+          'justify-center': flexJustify === 'center',
+          'justify-between': flexJustify === 'between',
+          'justify-around': flexJustify === 'around',
+          'justify-evenly': flexJustify === 'evenly',
+        },
+        containerClasses
```

Comment:
these are all 1-1 with the tailwind classes. I think just stick to one for now and just add when we need to cover other scenarios. this will just introduce more logic that we don't need right now. 

container classes we also don't need, we can just use the className prop.